projects/storefrontlib/src/cms-components/anonymous-consent-management/anonymous-consent-management-banner.component.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('AnonymousConsentManagementBannerComponent', () => {
       imports: [I18nTestingModule],
       declarations: [
         AnonymousConsentManagementBannerComponent,
-        MockFeatureLevelDirective,
+        MockFeatureDirective,
```

Comment:
If I recall, you created MockFeatureLevelDirective because of the feature-level, but do we still need it now?

Reply:
Correct. We can leave for the future use, someone might need it.